molecule/default/prepare.yml

```diff
@@ -0,0 +1,33 @@
+- name: Install nginx
+  hosts: all
+  # connection: local
+  become: no
+  gather_facts: yes
+  tasks:
+
+    - name: Update yum
+      yum:
+        update_cache: yes
+      when: ansible_os_family == 'RedHat'
+
+    - name: Ensure nginx is installed
+      package:
+        name:
+          - nginx
+        state: present
+      when: ansible_os_family == 'RedHat'
+
+    - name: Ensure python is installed
+      package:
+        name:
+          - python
+        state: present
+      when: ansible_os_family == 'RedHat'
+
+    - name: Ensure nginx is installed for Debian
+      apt: pkg=nginx state=present update_cache=true
+      when: ansible_os_family == 'Debian'
```

Comment:
If this task happens to fail, just update apt's cache first by itself before trying to install anything.